docs/latest/intro/compatibility.rst

```diff
@@ -7,13 +7,13 @@ can function together for Envoy Gateway. Supported version combinations
 are listed below; **bold** type indicates the versions of the Envoy Proxy
 and the Gateway API actually compiled into each Envoy Gateway release.
 
-+--------------------------+---------------------+---------------------+----------------------------+
-| Envoy Gateway version    | Envoy Proxy version | Gateway API version | Kubernetes version         |
-+--------------------------+---------------------+---------------------+----------------------------+
-| v0.2.0                   | **v1.23-latest**    | **v0.5.1**          | v1.24                      |
-+--------------------------+---------------------+---------------------+----------------------------+
-| latest                   | **dev-latest**      | **v0.5.1**          | v1.23.6, v1.24, v1.25      |
-+--------------------------+---------------------+---------------------+----------------------------+
++--------------------------+---------------------+---------------------+------------------------------+
+| Envoy Gateway version    | Envoy Proxy version | Gateway API version | Kubernetes version           |
++--------------------------+---------------------+---------------------+------------------------------+
+| v0.2.0                   | **v1.23-latest**    | **v0.5.1**          | v1.24, v1.26                 |
++--------------------------+---------------------+---------------------+------------------------------+
+| latest                   | **dev-latest**      | **v0.5.1**          | v1.23.6, v1.24, v1.25, v1.26 |
++--------------------------+---------------------+---------------------+------------------------------+
```

Comment:
No need to update this file since https://github.com/envoyproxy/gateway/pull/1002 does it.